app/views/resumes/show.html.haml

```diff
@@ -19,31 +19,28 @@
               Download PDF
 
       .container
-        .row
-          .col-md-5
-            = render 'resumes/sections/personal_details', resume: @resume
```

Comment:
removed the columns on the resume view. makes all items stack as oppose to being parallel to each other.

Reply:
It definitely looks much cleaner now but the thing is that it causes the details of the resume to blend into the background (the creation date and the edit buttons). My opinion, I would like to see if we can bring the card or a div element that looks like the card to encapsulate the resume and all of its details.

Reply:
so for the details what if we made it a color and font that contrasts better like for the last updated date we could do bold and white text. Thoughts?

Reply:
yea that works!